src/smart-escrow/Errors.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+/// @dev The error is thrown when an address is not set.
+error AddressIsZeroAddress();
```

Comment:
normally people just put these in the contract or interface itself

Reply:
I've seen this pattern as well, but I can definitely just put it in the contract if preferred!